addon/templates/components/frost-chart-svg-plot-grid.hbs

```diff
@@ -1,9 +1,11 @@
 {{! Template for the frost-chart-svg-plot-grid component }}
 
-{{#each _xAxisTicks as |tick|}}
-  <line x1={{tick.x}} y1={{tick.y}} x2={{tick.x}} y2=0 />
-{{/each}}
+{{frost-chart-svg-plot-x-grid
+  chartState=chartState
+  ticks=ticks
+}}
 
-{{#each _yAxisTicks as |tick|}}
-  <line x1=0 y1={{tick.y}} x2={{tick.x}} y2={{tick.y}} />
-{{/each}}
+{{frost-chart-svg-plot-y-grid
+  chartState=chartState
+  ticks=ticks
```

Comment:
I'm a little concerned that the abstraction for the grid is _slightly_ broken.  In particular, using the same `ticks` function for both x and y is likely to break down quickly if we provide a different `ticks` function to the split x and y axis.

We _could_ guard this by preferring an `xTicks` and `yTicks` while still allowing `ticks`, but I think it's probably best to mark the old `grid` as deprecated and remove it in the next major (or we could just release a major immediately, there isn't a large uptake yet)